src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  color: $main-accent;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+
+  &__image-block {
+    margin: auto;
+  }
```

Comment:
The `&__image-block` element is redundant since it only contains a margin property. The same result can be achieved by applying the margin directly to the `&__image` element, which will simplify the structure.